config/example_config.toml

```diff
@@ -12,7 +12,7 @@ max_vault_apy = 0.9  # as a decimal
 base_asset_price = 1  # aka market price
 
 [amm]
-pricing_model_name = "Hyperdrive" # specify a pricing model
+pricing_model_name = "Yieldspace" # specify a pricing model
```

Comment:
I think "Hyperdrive" is a better default. Happy to keep it like this until @sentilesdal finished us flat+curve testing though.

Reply:
oh actually I agree -- I changed this when hunting a bug and never changed it back.